atlasdb-cassandra-multinode-tests/src/test/java/com/palantir/cassandra/multinode/OneNodeDownMetadataTest.java

```diff
@@ -25,51 +24,43 @@
 import com.google.common.collect.ImmutableMap;
 import com.palantir.atlasdb.AtlasDbConstants;
 import com.palantir.atlasdb.keyvalue.api.TableReference;
-import com.palantir.atlasdb.table.description.ColumnMetadataDescription;
-import com.palantir.atlasdb.table.description.NameMetadataDescription;
-import com.palantir.atlasdb.table.description.TableMetadata;
-import com.palantir.atlasdb.transaction.api.ConflictHandler;
-import com.palantir.common.exception.AtlasDbDependencyException;
+import com.palantir.atlasdb.keyvalue.cassandra.CassandraKeyValueService;
 
-public class OneNodeDownMetadataTest {
+public class OneNodeDownMetadataTest extends AbstractDegradedClusterTest {
+    private static final TableReference TEST_TABLE_2 = TableReference.createWithEmptyNamespace("test_table_2");
+    private static final TableReference TEST_TABLE_3 = TableReference.createWithEmptyNamespace("test_table_3");
+
+    @Override
+    void testSetup(CassandraKeyValueService kvs) {
+        kvs.createTable(TEST_TABLE, AtlasDbConstants.GENERIC_TABLE_METADATA);
+        kvs.createTable(TEST_TABLE_2, AtlasDbConstants.EMPTY_TABLE_METADATA);
+        kvs.createTable(TEST_TABLE_3, AtlasDbConstants.EMPTY_TABLE_METADATA);
+    }
 
     @Test
     public void canGetMetadataForTable() {
-        byte[] metadata = OneNodeDownTestSuite.kvs.getMetadataForTable(OneNodeDownTestSuite.TEST_TABLE);
-        assertEquals(TableMetadata.BYTES_HYDRATOR.hydrateFromBytes(AtlasDbConstants.GENERIC_TABLE_METADATA),
-                TableMetadata.BYTES_HYDRATOR.hydrateFromBytes(metadata));
+        assertKvsReturnsGenericMetadata(TEST_TABLE);
     }
 
     @Test
     public void canGetMetadataForAll() {
-        Map<TableReference, byte[]> metadataMap = OneNodeDownTestSuite.kvs.getMetadataForTables();
-        assertEquals(TableMetadata.BYTES_HYDRATOR.hydrateFromBytes(AtlasDbConstants.GENERIC_TABLE_METADATA),
-                TableMetadata.BYTES_HYDRATOR.hydrateFromBytes(metadataMap.get(OneNodeDownTestSuite.TEST_TABLE)));
+        Map<TableReference, byte[]> metadataMap = getTestKvs().getMetadataForTables();
+        assertThat(getTestKvs().getMetadataForTables().get(TEST_TABLE))
+                .isEqualTo(AtlasDbConstants.GENERIC_TABLE_METADATA);
+        assertThat(metadataMap.get(TEST_TABLE)).isEqualTo(AtlasDbConstants.GENERIC_TABLE_METADATA);
```

Comment:
I'm confused - is there a reason we talk to the KVS twice, and if so, is there a reason we assert on the second call before the first?

Reply:
Got to test it twice, just to make sure! Or refactored but didn't remove the old code...